src/js/components/DropButton/DropButton.js

```diff
@@ -20,6 +20,7 @@ const DropButton = forwardRef(
       onClick,
       onClose,
       onOpen,
+      trapFocus = true,
```

Comment:
Do we need this? Since Drop already defaults `trapFocus` to `true`, can we let `dropProps` take care of overriding it if needed?

Reply:
yeah, i think you're right. just tested without it, works the same.